pandas/tests/scalar/period/test_period.py

```diff
@@ -212,58 +212,59 @@ def test_period_cons_combined(self):
         with tm.assert_raises_regex(ValueError, msg):
             Period('2011-01', freq='1D1W')
 
-    def test_timestamp_tz_arg(self):
-        for case in ['Europe/Brussels', 'Asia/Tokyo', 'US/Pacific']:
-            p = Period('1/1/2005', freq='M').to_timestamp(tz=case)
-            exp = Timestamp('1/1/2005', tz='UTC').tz_convert(case)
-            exp_zone = pytz.timezone(case).normalize(p)
-
-            assert p == exp
-            assert p.tz == exp_zone.tzinfo
-            assert p.tz == exp.tz
-
-            p = Period('1/1/2005', freq='3H').to_timestamp(tz=case)
-            exp = Timestamp('1/1/2005', tz='UTC').tz_convert(case)
-            exp_zone = pytz.timezone(case).normalize(p)
-
-            assert p == exp
-            assert p.tz == exp_zone.tzinfo
-            assert p.tz == exp.tz
-
-            p = Period('1/1/2005', freq='A').to_timestamp(freq='A', tz=case)
-            exp = Timestamp('31/12/2005', tz='UTC').tz_convert(case)
-            exp_zone = pytz.timezone(case).normalize(p)
-
-            assert p == exp
-            assert p.tz == exp_zone.tzinfo
-            assert p.tz == exp.tz
-
-            p = Period('1/1/2005', freq='A').to_timestamp(freq='3H', tz=case)
-            exp = Timestamp('1/1/2005', tz='UTC').tz_convert(case)
-            exp_zone = pytz.timezone(case).normalize(p)
-
-            assert p == exp
-            assert p.tz == exp_zone.tzinfo
-            assert p.tz == exp.tz
-
-    def test_timestamp_tz_arg_dateutil(self):
+    @pytest.mark.parametrize('case', ['Europe/Brussels',
+                                      'Asia/Tokyo', 'US/Pacific'])
+    def test_timestamp_tz_arg(self, case):
+        p = Period('1/1/2005', freq='M').to_timestamp(tz=case)
+        exp = Timestamp('1/1/2005', tz='UTC').tz_convert(case)
+        exp_zone = pytz.timezone(case).normalize(p)
+
+        assert p == exp
+        assert p.tz == exp_zone.tzinfo
+        assert p.tz == exp.tz
+
+        p = Period('1/1/2005', freq='3H').to_timestamp(tz=case)
+        exp = Timestamp('1/1/2005', tz='UTC').tz_convert(case)
+        exp_zone = pytz.timezone(case).normalize(p)
+
+        assert p == exp
+        assert p.tz == exp_zone.tzinfo
+        assert p.tz == exp.tz
+
+        p = Period('1/1/2005', freq='A').to_timestamp(freq='A', tz=case)
+        exp = Timestamp('31/12/2005', tz='UTC').tz_convert(case)
+        exp_zone = pytz.timezone(case).normalize(p)
+
+        assert p == exp
+        assert p.tz == exp_zone.tzinfo
+        assert p.tz == exp.tz
+
+        p = Period('1/1/2005', freq='A').to_timestamp(freq='3H', tz=case)
+        exp = Timestamp('1/1/2005', tz='UTC').tz_convert(case)
+        exp_zone = pytz.timezone(case).normalize(p)
+
+        assert p == exp
+        assert p.tz == exp_zone.tzinfo
+        assert p.tz == exp.tz
+
+    @pytest.mark.parametrize('case', ['dateutil/Europe/Brussels',
```

Comment:
same

Reply:
There's already a `tz` variable in the method; `case` is the name used for this in the status quo.

Reply:
pls change. this is really confusing.